File: src/sys/run_test_suite/tests/long_running_test.cc
// Copyright 2020 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include <stdio.h>

#include "gtest/gtest.h"

TEST(LongRunningTest, LongRunning) { sleep(10); }

File: src/sys/run_test_suite/tests/long_running_test.rs
// Copyright 2022 The Fuchsia Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#[fuchsia::test]
async fn long_running() {
    fuchsia_async::Timer::new(std::time::Duration::from_secs(10)).await;
}
